code/modules/clothing/factions/gezena.dm

```diff
@@ -130,14 +135,19 @@
 	desc = "The standard cap of the PGF military, in Navy colors. “betzu-il”, translating to “sun-blocker”, refers to the flap at the back for protection against natural hazards such as sunburns, sandstorms, and biting insects."
 	icon_state = "navalflap"
 	item_state = "bluecloth"
+	armor = list("melee" = 5, "bullet" = 5, "laser" = 5, "energy" = 5, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50) //praticality for enviro prot
```

Comment:
No armor on soft clothing items

Reply:
It had armor before - do you want it gone because it provides enviro?

Reply:
Armor as in melee etc